packages/core/.projen/tasks.json

```diff
@@ -211,14 +181,6 @@
         }
       ]
     },
-    "test:app": {
```

Comment:
This is run as a separate pipeline step now to benefit from turborepo caching once we're able to optimize the configuration.